_data/sites.json

```
@@ -8802,6 +8802,17 @@
         ]
     },
 
+    {
+        "name": "Pennsylvania Turnpike (Toll By Plate)",
+        "url": "https://www.paturnpike.com/help-center#contact-form",
+        "difficulty": "hard",
+        "notes": "NOT FOR EZPASS ACCOUNTS. Fill out the form using the same information that is associated to your account. Alternatively, the account can be closed by calling 877-736-6727.",
+        "domains": [
+            "paturnpike.com",
+            "www.ezpass.csc.paturnpike.com"
```

```suggestion
            "paturnpike.com"
```
I don't think the second domain is needed